apps/frontend/src/app/5_pages/ZeroPage/ZeroPage.tsx

```diff
@@ -234,7 +235,7 @@ export const ZeroPage: FC = () => {
                 <DialogBody>
                   <CloseCreditLine
                     onSubmit={handleTroveClose}
```

Comment:
Are you sure `handleTroveClose` can use `DLLR` to close an LoC. Test it pls and consult with julio/Tyrone/light if needed.

Reply:
@tiltom as I know, we can't close it with DLLR atm but @creed-victor will add the functionality for this. Victor pls confirm it

Reply:
@pietro-maximoff that's true, I just confirmed with @creed-victor it will be done in https://sovryn.atlassian.net/browse/SOV-1578 so I am merging this PR and closing the story.